pkg/api/v1beta1/defaults.go

```diff
@@ -102,3 +108,14 @@ func init() {
 		},
 	)
 }
+
+// With host networking default all container ports to host ports.
+func defaultHostNetworkPorts(containers *[]Container) {
+	for i := range *containers {
+		for j := range (*containers)[i].Ports {
+			if (*containers)[i].Ports[j].ContainerPort == 0 {
+				(*containers)[i].Ports[j].ContainerPort = (*containers)[i].Ports[j].HostPort
```

Comment:
We used HostPort, because user has to specify what HostIP, they are using with the Port.
They wont specify ContainerIP.
Since HostPort goes well with HostIP, we used that instead of using ContainerIP, though both HostPort and ContainerPort are really the same in the net=host case.